### PR TITLE
Align extra mouse button numbering on Linux, Mac and Windows

### PR DESCRIPTION
**Pull request by lkishalmi**
Proposal for aligning the extra mouse button codes on Linux with Windows, and Mac.

### CODE REVIEW

**Review by neilcsmith-net — CHANGES_REQUESTED** [review]
Sorry, been tied up the last few days, but just been looking at how to fix this myself.  Thanks for taking a look.  Really think it should be in the event handler itself though.

File: platform/openide.util.ui/src/org/openide/util/Utilities.java [hunk 1]
```diff
@@ -850,7 +854,14 @@ public static int mouseWheelDownKeyCode() {
      */
     public static int mouseButtonKeyCode(int button) {
         if (button >= 4 && button < 10) {
-            return 0x292 + (button - 1);
+            if (getOperatingSystem() == OS_LINUX) {
```
[the remark this answers is not in the document]
Comment (lkishalmi) [reply platform/openide.util.ui/src/org/openide/util/Utilities.java:857]: I see your point.  Shift + wheel up/down works as horizontal scrolls in Java, however those are not generating button 4/5 events at least  on Linux.

[the remark this answers is not in the document]
Comment (neilcsmith-net) [reply platform/openide.util.ui/src/org/openide/util/Utilities.java:857]: Yes, as linked in my original PR, they're dropped in the JDK at https://github.com/openjdk/jdk/blob/master/src/java.desktop/unix/classes/sun/awt/X11/XWindow.java#L731

**Comment by lkishalmi** [comment]
Closing in favor of #6644

**Pull request closed without merging**